src/foam/nanos/notification/sms/TwilioSMSService.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.notification.sms',
+  name: 'TwilioSMSService',
+
+  implements: [
+    'foam.nanos.notification.sms.SMSService'
+  ],
+
+  documentation: 'Sends SMS message through Twilio',
+
+  javaImports: [
+    'com.twilio.Twilio',
+    'com.twilio.rest.api.v2010.account.Message',
+    'com.twilio.type.PhoneNumber',
+    'foam.dao.DAO',
+    'foam.nanos.app.TwilioConfig',
+    'foam.nanos.notification.sms.SMSStatus',
+    'foam.util.SafetyUtil'
+  ],
+
+  methods: [
+    {
+      name: 'sendSms',
```

Comment:
Can you add OMs similar SMTPservice please then we can alarm on congestion and failure.

Reply:
Guessing you meant PMs, ya sure